Jellyfin.Plugin.PushOverNotifications/Api/ServerApiEntryPoints.cs

```diff
@@ -21,11 +20,11 @@ public class TestNotification : IReturnVoid
     class ServerApiEndpoints : IService
     {
         private readonly IHttpClient _httpClient;
-        private readonly ILogger _logger;
+        private readonly ILogger IloggerFactory;
```

Comment:
nonono, sorry this was not what I meant. I have been busy with work so I'm sorry that I didn't reply in time..